### PR TITLE
Move voters.json to server directory

### PR DESCRIPTION
**Pull request by yet-another-account**
This way, we can view the json in-browser

### CODE REVIEW

**Comment by reddraggone9** [comment]
@abrad1212 If you're still around, could you take a look at this and cast a vote?

**Comment by PlasmaPower** [comment]
I think `voters.json` currently exists as `cron/voters.json`, not in the root.

File: cron/poll_pull_requests.py [hunk 1]
```diff
@@ -92,13 +92,13 @@ def poll_pull_requests():
         # This sets up a voting record, with each user having a count of votes
         # that they have cast.
         try:
-            fp = open('voters.json', 'x')
+            fp = open('server/voters.json', 'x')
```
Comment (PlasmaPower) [inline cron/poll_pull_requests.py:95]: Should be `../server/voters.json`

File: cron/poll_pull_requests.py [hunk 2]
```diff
@@ -92,13 +92,13 @@ def poll_pull_requests():
...
             fp.close()
         except:
             # file already exists, which is what we want
             pass
 
-        with open('voters.json', 'r+') as fp:
+        with open('server/voters.json', 'r+') as fp:
```
Comment (PlasmaPower) [inline cron/poll_pull_requests.py:101]: Should be `../server/voters.json`

File: chaos.py [hunk 3]
```diff
@@ -34,6 +35,10 @@ def main():
 
     log = logging.getLogger("chaosbot")
 
+    if exists("voters.json"):
+        log.info("Moving voters,json to server directory!")
+        shutil.move("./voters.json", "./server/voters.json")
```
Comment (PlasmaPower) [inline chaos.py:40]: Should be `./cron/voters.json`, not `./voters.json`.

**Review by PlasmaPower — CHANGES_REQUESTED** [review]

File: chaos.py (hunk 3 above)
Comment (PlasmaPower) [reply chaos.py:40]: Also slight typo in the log: `voters.json` not `voters,json`.

**Comment by reddraggone9** [comment]
@PlasmaPower 
> I think `voters.json` currently exists as `cron/voters.json`, not in the root.
```
python3 -c 'help(open)'
```
> file is either a text or byte string giving the name (and the path
    if the file isn't in the **current working directory**)

What makes you think the current working directory is cron?

**Comment by PlasmaPower** [comment]
Oops, I was looking at the `execl` code right below it which was using `THIS_DIR`. Perhaps this should too? I don't like relying on the CD very much.

**Review by PlasmaPower — APPROVED** [review]

**Comment by reddraggone9** [comment]
Yeah, that'd probably be a good idea. I'd like to redo everything using pathlib at some point. I could also make everything repo root/`__file__` relative at that point.

**Pull request merged**